### DISCUSSION TITLE
eTag

### DISCUSSION THREAD

**Discussion by binaryk** [binaryk/laravel-restify#397] [Ideas]
https://hergen.nl/caching-your-laravel-api-with-etag-and-conditional-requests?ref=laravelnews

**Comment by cristijora** [comment]
I think this is a good idea to improve performance, however we should consider a few things when caching data
- Cached data could be invalidated by many things like
 - POST, PUT, PATCH, DELETE requests
 - Async operations like Actions or Jobs
 
 Also if we use it for reducing bandwidth, it might have some implications on the frontend implementations that we have to also test and make sure it's backwards compatible.
